coc.md

```diff
@@ -4,81 +4,72 @@ description: Guidelines on how we expect our members to behave.
+### What we are
 
-## Non-Goals
+Fosstodon is a Mastodon instance that federates with many other servers in the fediverse, also known as the open social web. The theme of this instance is Free/Libre/Open-Source software and adjacent communities. This is what the "FOSS" in our name represents.
 
-While it's important to know what Fosstodon is, it's just as important to understand what we're not. Fosstodon is **not**:
+### Intentions
 
-* A free speech zone. Fosstodon is a community with specific — and high — standards of behaviour. If you would like to exercise your full free speech rights, you can do so **somewhere else**; while on Fosstodon, you should agree to follow the community rules set out in this CoC.
-* An advocacy space. There are many important, controversial and political issues; and advocating for what you believe is important. However, Fosstodon is not the place for that advocacy. Even correct views can violate our CoC.
-* An experts-only space. We all share a love of FOSS, but no Fosstodon user should be shamed for not understanding a basic concept or told to read the manual before asking questions about a certain technology.
+Fosstodon intends to provide a friendly environment for those who want a social media experience that prioritizes FOSS, where you can share your interests, passions, and achievements. Nobody should feel unwelcome on the basis of their skills, experience, or ability to read a manual; Fosstodon is *not* an experts-only space.
 
-## Public Toots
+While using Fosstodon or engaging with Fosstodon members, everyone should be safe from harassment and similar antisocial behavior. As a community, we aim to curate a safe place for anyone, regardless of their characteristics including but not limited to: race, ethnicity, national identity, spoken language, sex, gender identity or expression, sexuality, sexual orientation, physical or mental health status, disability, other body characteristics, or age.
 
-Public Toots are those that are posted to the public timelines. This means that anyone on Fosstodon can see your Toot on the local timeline and anyone on the Internet can see your Toot via the [see what's happening](https://fosstodon.org/public) page. This also means that your Toot will be visible to any instances that Fosstodon federates with, via the federated timeline.
+Additionally, as members, you are expected to practice tolerance of others' expressions of faith, personal philosophy, or political affiliation, *provided* those values do not conflict with this code of conduct or impact the safety of others. We do not want a [paradox of tolerance](https://en.wikipedia.org/wiki/Paradox_of_tolerance) on Fosstodon.
 
-## Non-Public Toots
+We understand that many members have strong opinions about "freedom of speech", however this is not a feature we offer.
```

Comment:
It's not entirely clear to me what is implied by "a feature" in this sentence, it may help to be explicit, especially given that "freedom of speech" is in quotes? I think what you're saying is that Fosstodon does not permit any member to say anything they want. (Perhaps other servers do, and tout it as a feature? I don't actually know, I'm just guessing based off that wording)

Reply:
It seems to be duplicating the "free speech zone" restriction above and could probably be safely deleted:
```suggestion
```

Reply:
There is a whole subset of servers in the fediverse that are centered around the notion of free speech, yes.

It's also common for folks to conflate the concepts of free speech with the values of free as in freedom in the FOSS world, and expect free software related communities and services to accommodate it accordingly.

Between these two datapoints, I think it is necessary to emphasize that this is not a place for free speech even if members feel it should be. Phrasing it as a "feature" felt thematic to software and provides a bit of levity for the otherwise heavy document.